dev/com.ibm.ws.jca.cm/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -22,6 +22,7 @@
 
  <OCD id="com.ibm.ws.jca.connectionManager" name="%connectionManager" description="%connectionManager.desc" ibm:alias="connectionManager" ibmui:localization="OSGI-INF/l10n/metatype">
   <AD id="agedTimeout"                        name="%agedTimeout"  description="%agedTimeout.desc"  default="-1"     type="String"  ibm:type="duration(s)" min="-1"/>
+  <AD id="abortLongRunningInuseConnections"   name="abortLongRunningInuseConnections"  description="%abortLongRunningInuseConnections.desc"  default="-1"     type="String"  ibm:type="duration(s)" min="-1"/>
```

Comment:
Consider changing to minutes `ibm:type="duration(m)"` in order to prevent the configuration of values like 1 second, 5 seconds, or 10 seconds.